src/components/admin/companies/grid/CreateForm.tsx

```diff
@@ -1,5 +1,9 @@
 import React from 'react'
 import * as yup from 'yup'
+import {
+  calculateChecksumForNineDigitsEIK,
```

Comment:
I'd rename those to: validateChecksumForNineDigitsEIK

or may even something simpler like: validateEIK9

Reply:
functions have been renamed